pages/common/bfs.md

```diff
@@ -0,0 +1,36 @@
+# bfs
+
+> Breadth-first search for your files.
+> More information: <https://manned.org/bfs>.
+
+- Find files by extension:
+
+`bfs {{root_path}} -name '{{*.ext}}'`
+
+- Find files matching multiple path/name patterns:
+
+`bfs {{root_path}} -path '{{**/path/**/*.ext}}' -or -name '{{*pattern*}}'`
```

Comment:
```suggestion
`bfs {{root_path}} -path '{{**/path/**/*.ext}}' -or -name '{{pattern}}'`
```
What's the point of the * symbols in "pattern"?
if they were meant to make the text **bold** or _italics_, it is not supported.

Reply:
It means substring as in anywhere in the text of name has pattern *wildcard from start - *wildcard to end etc

Reply:
Thanks for explaining.